test/Rench_Test/ChildProcessSpawnTest.re

```diff
@@ -31,7 +31,7 @@ describe("ChildProcess", ({describe, _}) => {
       let proc = ChildProcess.spawn("node", [|"-e", script|]);
 
       let data = ref("");
-      let _event =
```

Comment:
This was just plain wrong, and is another good reason why you should add proper type annotations to ignored values, as a sanity check for yourself. That is what a static type system is for, after all.